run/idp-sql/app.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "idp-sql",
+  "env": {
+    "DB_PASSWORD": {
+      "description": "postgreSQL password",
+      "value": "password1234"
```

Comment:
This will prompt the user to enter a password for the creation of the Cloud SQL instance. I will remove the default so people won't just rely on my super secret password.